src/components/fee-summary/fee.ts

```diff
@@ -0,0 +1,8 @@
+export interface Fee {
+  fee: number
+  filingType: string
+  priorityFees: number
+  futureEffectiveFees: number
+  serviceFees: number
+  total: number
+}
```

Comment:
Why not move this (and the other) interfaces to the interfaces package?

Reply:
while moving the interfaces over to the interfaces package I noticed that the FilingData.ts and the existing filing-data-interface.ts are very similar. which one should I use?

Reply:
They don't look like the same object to me.

However, your 2 interfaces look similar. Are they the same object (used for the same purpose)?
- Fee
- PayData